File: Exercise/Hard/06. Arithmetic Expression Evaluator/Code/Arithmetic Expression Evaluator.ts
function EvaluateExpression(expression: string): any {
    try {
        return eval(expression); 
    } catch (error) {
        console.error(`Error: ${error.message}`);
        return null;
    }
}
console.log(EvaluateExpression("(3 + 5 * 2 + 1) / 2"));

File: Exercise/Hard/06. Arithmetic Expression Evaluator/Code/Arithmetic Expression Evaluator.py
def evaluate_expression(expression):
    try:
        return eval(expression)  
    except Exception as e:
        print(f"Error: {e}")
        return None

def main():
    expression = input()
    result = evaluate_expression(expression)
    if result is not None:
        print(result)

if __name__ == "__main__":
    main()
